schema/artist/index.js

```diff
@@ -132,6 +138,15 @@ const ArtistType = new GraphQLObjectType({
               forsale_artworks_count),
             partner_shows: numeral(({ id }) =>
               total(`related/shows`, { artist_id: id })),
+            related_artists: numeral(({ id }) =>
+              total(`related/layer/main/artists`, { artist: id })),
+            articles: numeral(({ _id }) =>
+              positron('articles', defaults({
+                artist_id: _id,
+                published: true,
+                size: 0,
+                total_count: 1,
+              })).then(({ results }) => results.length)),
```

Comment:
It doesn’t seem like you’re actually using the response header but instead just counting the number of articles. Does positron support the same pagination parameters as gravity at all?